### ISSUE TITLE
[1.8.9] Redcrystal does not update when receiving a weak redstone signal

### ISSUE THREAD

**Issue by Xiaminou** [Tuhljin/Automagy#47]
![2016-05-04_16 10 01](https://cloud.githubusercontent.com/assets/8670392/15016722/ce8fdf88-1212-11e6-9c86-20a97fa90e12.png)
![2016-05-04_16 10 16](https://cloud.githubusercontent.com/assets/8670392/15016730/d5a24b9e-1212-11e6-9634-adcd8d5f8e0c.png)
![2016-05-04_16 09 06](https://cloud.githubusercontent.com/assets/8670392/15016735/dc124ec0-1212-11e6-88fa-daff9e40f1f0.png)

In this example I have a drawer with a redstone upgrade (screenshot1) emitting a strong redstone signal (level depending on the number of items it contains) into the bricks below which in turn emit a weak redstone signal to the redcrystal (screenshot 3). As you can see the redcrystal (screenshot 3) is still emitting 15 redstone from when the drawer was full, however the redcrystal (screenshot 2) is correctly emitting for the new number of items.

**Comment by Tuhljin** [comment]
I changed something related to this for the next release. Please test it out with the new version once it's available and let me know if the problem is solved. Thanks.

**Comment by Xiaminou** [comment]
When is the next release?

**Comment by Tuhljin** [comment]
Today.

**Comment by Xiaminou** [comment]
not fixed.

**Comment by Tuhljin** [comment]
It's the block emitting a signal's responsibility to send block updates to surrounding blocks. Does regular redstone pick up the signal as it changes?

**Comment by Xiaminou** [comment]
Good call, the redstone doesn't update either, I'm going to report to Storage Drawers

**Comment by jaquadro** [comment]
This should be fixed in the latest SD releases for all game versions.
